res/index.min.js

```diff
@@ -1 +1 @@
snappingColumns(a).snappingRows(u),s.forEach((function(r,s){return t.__awaiter(this,void 0,void 0,(function(){var s,n,a,u,l,p,c=this;return t.__generator(this,(function(h){switch(h.label){case 0:if(!(s=e.grid.getContent(r.id()))){for(n=0,a=0,3,s=r.createWidget();null!==e.grid.getCell(3*n,3*a);)++a>=o&&(a=0,++n);e.grid.setContent(3*n,3*a,s,null,3,3)}switch(s.classID()){case"observable-md_ObservableMD":case"observable-md_Observable":return[3,1];case"graph_Graph":return[3,4]}return[3,5];case 1:return[4,i.Workunit.attach({baseUrl:e._espWorkunit._protocol+"//"+e._espWorkunit._host},e._espWorkunit._wuid)];case 2:return[4,(u=h.sent()).fetchResults()];case 3:return l=h.sent(),p={wuid:function(){return u.Wuid},outputs:function(){return l.map((function(e){return{name:e.Name,count:e.Total,value:e.Value,logicalFile:e.LogicalFileName}}))}},l.forEach((function(e){p[e.Name]=function(){return t.__awaiter(c,void 0,void 0,(function(){return t.__generator(this,(function(t){return[2,e.fetchRows()]}))}))}})),s.plugins(p),[3,6];case 4:return s.on("vertex_click",(function(t,r,i){e.refreshFilters(this.id(),t,r,i)})),[3,6];case 5:s.on("click",(function(t,r,i){e.refreshFilters(this.id(),t,r,i)})),h.label=6;case 6:return s._wuMeta=r,r.widget(s),e._wuWidgets.push(r),e._wuWidgetMap[r.id()]=r,[2]}}))}))})),e.grid}))},e.prototype.refresh=function(){var e=this;this._wuWidgets.forEach((function(t){t.refreshData(e._wuDashSel,e.grid.maxRowCount())}))},e.prototype.refreshFilters=function(e,t,r,i){var s=this;i?this._wuDashSel[e]={row:t,col:r}:delete this._wuDashSel[e],this._wuWidgets.forEach((function(t){t.filteredBy().forEach((function(r){e===r.source+f&&t.refreshData(s._wuDashSel,s.grid.maxRowCount())}))}))},e}(),w=function(e){function r(){return e.call(this)||this}return t.__extends(r,e),r.prototype.toggleProperties=function(){var e=s.Dermatology.prototype.toggleProperties.apply(this,arguments);return this._showProperties||this.wuDashboard.submitPersist(),e},r.prototype.reset=function(){var e=this;this.wuDashboard.resetPersist().then((function(t){a.cache({}),delete e._prevEspUrl,e.render((function(e){}))}))},r.prototype.enter=function(){s.Dermatology.prototype.enter.apply(this,arguments);var e=this;this._resetButton=(new o.Button).id(this.id()+"_reset").value("Reset").on("click",(function(){e.reset()})),this._toolbar.widgets([this._resetButton,this._propsButton])},r.prototype.update=function(){if(s.Dermatology.prototype.update.apply(this,arguments),this._prevEspUrl!==this.espUrl()){this._prevEspUrl=this.espUrl(),this.espCache()&&a.cache(JSON.parse(this.espCache()));var e=this;this.wuDashboard=new m(this.espUrl()),this.wuDashboard.createGrid().then((function(t){e.widget(t).render((function(t){e.wuDashboard.refresh()}))}))}},r}(s.Dermatology);w.prototype._class+=" BundleDermatology",w.prototype.publish("espUrl",null,"string","ESP Url"),w.prototype.publish("espCache",null,"string","ESP Cache"),e.BundleDermatology=w,e.WUDashboard=m,Object.defineProperty(e,"__esModule",{value:!0})}));
```

Comment:
This looks rather like generated code to me - if so, should it really be checked in?

Reply:
Yes - its part of the bundle and this is where `ecl bundle` pulls the sources from...